website/blog/2020-11-06-sbt-BSP-support.md

```diff
@@ -0,0 +1,147 @@
+---
+author: Chris Kipp
+title: sbt BSP support
+authorURL: https://twitter.com/ckipp01
+authorImageURL: https://avatars1.githubusercontent.com/u/13974112?s=400&u=7b6a2ddab8eec6f99e4e40ae9b81f71cb5ba92e5&v=4
+---
+
+If you've been following the [sbt
+releases](https://github.com/sbt/sbt/releases), 1.4.x introduced some great new
+features, one of those being BSP support. This effort was initiated by community
+members and proposed to the [Scala Center Advisory
+Board.](https://github.com/scalacenter/advisoryboard/blob/master/proposals/023-bsp.md).
+Then thanks to the work of [Adrien Piquerez](https://twitter.com/adrienpi2) and
+[Eugene Yokota ](https://twitter.com/eed3si9n) BSP support became a reality in
+1.4.0. You can read a more detailed account of the reason behind the work and
+some of the details of the implementation in this blog post, [BSP Support in sbt
+1.4](https://www.scala-lang.org/blog/2020/10/27/bsp-in-sbt.html).
+
+The Metals team is also happy to announce that as of 0.9.5, the process to use
+other BSP servers like sbt is now much smoother. In some ways, BSP is an
+implementation detail abstracted away from a user, and you may not directly
+interact at all with your build server, even though it's pivotal for the Metals
+experience. Therefore, we'd like to answer some common questions about what this
+means to you as a user, explain some of the default choices Metals makes for
+you, and show you how to explore the sbt BSP server.
+
+## What is BSP?
+
+First off, you may be wondering what BSP is. The [Build Server
+Protocol](https://build-server-protocol.github.io/) (BSP) is a _Protocol for IDEs and
+build tools to communicate about compilation, running, testing, debugging, and
+much more._ If you're familiar with the [Language Server
+Protocol](https://microsoft.github.io/language-server-protocol/) (LSP), BSP is
+complementary to LSP and inspired by it. Where LSP allows your editor to
+abstract over various programming languages by a shared way to communicate to a
+language server, BSP allows IDE's to abstract over various build servers. A
+typical example of this can be illustrated like so:
+
+![LSP BSP example](https://i.imgur.com/0RRUDlU.png)
```

Comment:
Could you maybe add titles to each icon?

Reply:
Yea sure, I added them in before each image.